tests/integration/targets/aci_l3out_extepg_to_route_control_profile/tasks/main.yml

```diff
@@ -0,0 +1,196 @@
+
+- name: Add a new l3out
+  aci_l3out:
+    <<: *aci_info
+    tenant: ansible_tenant
+    name: ansible_l3out
+    description: l3out for Ansible tenant
+    domain: ansible_dom
+    route_control: export
+    vrf: ansible_vrf
+    l3protocol: ospf
+    state: present
+
+- name: Add a new ExtEpg
+  aci_l3out_extepg:
+    <<: *aci_info
+    tenant: ansible_tenant
+    l3out: ansible_l3out
+    name: ansible_extEpg
+    description: ExtEpg for Ansible l3out
+    state: present
+
+- name: Add route control profile - ansible_rtctrl_profile
+  cisco.aci.aci_route_control_profile:
+    <<: *aci_info
+    tenant: ansible_tenant
+    l3out: ansible_l3out
+    profile: ansible_rtctrl_profile
+    prof_type: global
+    state: present
+
+- name: Add route control profile - ansible_rtctrl_profile2
+  cisco.aci.aci_route_control_profile:
+    <<: *aci_info
+    tenant: ansible_tenant
+    l3out: ansible_l3out
+    profile: ansible_rtctrl_profile2
+    prof_type: global
+    state: present
+
+# BEGIN WITH TESTS
+- name: Add route control profile binding (check_mode)
+  cisco.aci.aci_l3out_extepg_to_route_control_profile: &binding_present
+    <<: *aci_info
+    tenant: ansible_tenant
+    l3out: ansible_l3out
+    ext_epg: ansible_extEpg
+    profile: ansible_rtctrl_profile
+    direction: import
+    state: present
+  check_mode: yes
+  register: cm_add_binding
+
+- name: Add route control profile binding (normal mode)
+  cisco.aci.aci_l3out_extepg_to_route_control_profile:
+    <<: *binding_present
+  register: nm_add_binding
+
+- name: Verify add route control profile binding
+  assert:
+    that:
+    - cm_add_binding is changed
+    - nm_add_binding is changed
+    - cm_add_binding.previous == nm_add_binding.previous == []
+    - cm_add_binding.sent.l3extRsInstPToProfile.attributes.direction == nm_add_binding.sent.l3extRsInstPToProfile.attributes.direction == 'import'
+    - cm_add_binding.sent.l3extRsInstPToProfile.attributes.tnRtctrlProfileName == nm_add_binding.sent.l3extRsInstPToProfile.attributes.tnRtctrlProfileName == 'ansible_rtctrl_profile'
+
+- name: Add route control profile binding again, check if idempotency works
+  cisco.aci.aci_l3out_extepg_to_route_control_profile:
+    <<: *binding_present
+  register: add_binding_again
+
+- name: Verify add_binding_again
+  assert:
+    that:
+    - add_binding_again is not changed
+
+# UPDATE BINDING
+- name: Change route control profile binding
+  cisco.aci.aci_l3out_extepg_to_route_control_profile: &binding_update
+    <<: *binding_present
+    profile: ansible_rtctrl_profile2
+  register: update_binding
+
+- name: Verify update_binding
+  assert:
+    that:
+    - update_binding is changed
+    - update_binding.sent.l3extRsInstPToProfile.attributes.tnRtctrlProfileName == 'ansible_rtctrl_profile2'
+    - update_binding.current.0.l3extRsInstPToProfile.attributes.tnRtctrlProfileName == 'ansible_rtctrl_profile2'
+
+# QUERY ALL BINDINGS
+- name: Query all route control profile bindings
```

Comment:
should we add more route control profile bindings to assure all are indeed found?

Reply:
This applies for the other test files also.

Reply:
added